ServerCore/Pages/Teams/AllTeams.cshtml

```diff
@@ -53,6 +56,16 @@ Registered Teams: <b>@Model.Teams.Count/@Model.Event.MaxNumberOfTeams</b>, Regis
                         <text>Disqualified</text>
                     }
                     </td>
+                    <td>
+                        if (item.IsRemoteTeam)
```

Comment:
Is this OK without an @? If not, why not change it to this nicer format from elsewhere in this PR:

```
@item.Team.IsRemoteTeam ? Remote : Local
```

Reply:
![image](https://github.com/user-attachments/assets/46c98b40-b5b2-421e-a214-db7fa43275e0)
Alternatively, it could be that that's not actually the page that any uses and it's been broken for 6 years. Working on editing the correct page now.

Reply:
And also fixing that ternary statement that I could have sworn I fixed already (must have lost a change somewhere)